apps/cli/src/transforms/links.ts

```diff
@@ -96,13 +101,20 @@ function mutateStaticLink(session: ISession, link: GenericNode, linkFile: string
   link.static = true;
 }
 
-export function transformLinks(session: ISession, mdast: Root, file: string) {
+const limitOutgoingConnections = pLimit(25);
+
+export async function transformLinks(
+  session: ISession,
+  file: string,
+  mdast: Root,
+  opts?: { checkLinks?: boolean },
+): Promise<string[]> {
   const links = selectAll('link,linkBlock', mdast) as GenericNode[];
-  links.forEach((link) => {
+  links.map((link) => {
```

Comment:
Just curious - what's the reasoning for this change? Naively, seems equivalent to me with in-place operations that don't return anything.

Reply:
Made this error in future:
![image](https://user-images.githubusercontent.com/913249/181045515-dbf266df-8e16-4d9d-a17c-f5d797e1e82c.png)